target/linux/mvebu/files-4.14/arch/arm/boot/dts/armada-385-wd-mcex2u.dts

```diff
@@ -0,0 +1,419 @@
+/*
+ * Device Tree file for Marvell Armada 385 evaluation board
+ * (DB-88F6820)
```

Comment:
I see You based this dts on upstream, it should mention this or remove this note.